src/TypeScript/EslintBridgeClient/Resources.resx

```diff
@@ -129,21 +129,21 @@
 {1}</value>
   </data>
   <data name="INFO_ServerAlreadyTerminated" xml:space="preserve">
-    <value>[eslint-bridge] EslintBridge process has already been terminated.</value>
+    <value>[eslint-bridge] Cannot terminate eslint-bridge process: no process is running.</value>
   </data>
   <data name="INFO_ServerHasExisted" xml:space="preserve">
     <value>[eslint-bridge] Server process HasExited: {0}</value>
   </data>
   <data name="INFO_ServerProcessId" xml:space="preserve">
-    <value>[eslint-bridge] Started eslint-bridge server. Process id: {0}</value>
+    <value>[eslint-bridge] Started eslint-bridge process. Process id: {0}</value>
   </data>
   <data name="INFO_ServerStarted" xml:space="preserve">
     <value>[eslint-bridge] Server running on port: {0}</value>
   </data>
   <data name="INFO_ServerTerminated" xml:space="preserve">
-    <value>[eslint-bridge] EslintBridge process terminated.</value>
+    <value>[eslint-bridge] Terminated eslint-bridge process. Process id: {0}</value>
```

Comment:
Just some string changes, to be consistent with the "eslint-bridge process" terminology.